beacon/goclient/aggregator.go

```diff
@@ -92,7 +92,7 @@ func isAggregator(committeeCount uint64, slotSig []byte) (bool, error) {
 
 // waitOneThirdOrValidBlock waits until one-third of the slot has transpired (SECONDS_PER_SLOT / 3 seconds after the start of slot)
 func (gc *goClient) waitToSlotTwoThirds(slot phase0.Slot) {
-	oneThird := gc.network.SlotDurationSec() / 3 /* one third of slot duration */
+	oneThird := gc.network.SlotDuration() / 3 /* one third of slot duration */
```

Comment:
why this change?

Reply:
It returns a `time.Duration`, not seconds. But this still may be changed depending on what changes I'll do in the future